interactions/client/context.py

```diff
@@ -41,11 +43,8 @@ class _Context(ClientSerializerMixin):
     """
 
     message: Optional[Message] = field(converter=Message, default=None, add_client=True)
-    author: Member = field(converter=Member, default=None, add_client=True)
-    member: Member = field(converter=Member, add_client=True)
+    member: Member = field(default=None, converter=Member, add_client=True)  # DMs?
```

Comment:
if member is none in the dms should we make it like `Optional[Member]`?

Reply:
Sure if you could add a suggestion for it?

Reply:
sure